xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -28,7 +26,7 @@ public protocol SphereProtocol {
     
     func readHeaderNames(slashlink: Slashlink) async -> [String]
     
-    func read(slashlink: Slashlink) async throws -> Memo
+    func read(slashlink: Slashlink) async throws -> MemoData
```

Comment:
Make type concrete. We were not using this customizable associatedtype anywhere.